merlin/data/celery/app.yaml

```diff
@@ -1,5 +1,8 @@
 celery:
```

Comment:
Has this been tested without the celery block?

Reply:
Yep, with no celery block this works too.